packages/www/pages/app/stream/[id].tsx

```diff
@@ -305,7 +305,7 @@ const ID = () => {
   const playbackId = (stream || {}).playbackId || "";
   const domain = isStaging() ? "monster" : "com";
   const globalIngestUrl = `rtmp://rtmp.livepeer.${domain}/live`;
-  const globalPlaybackUrl = `https://cdn.livepeer.${domain}/hls/${playbackId}/index.m3u8`;
+  const globalPlaybackUrl = `https://livepeercdn.${domain}/hls/${playbackId}/index.m3u8`;
```

Comment:
Do we have `livepeercdn.monster` as well? We probably should cc @iameli

Reply:
no, we're still using `cdn.livepeer.monster`